packages/theme-check-common/src/checks/unrecognized-render-snippet-params/index.ts

```diff
@@ -69,9 +69,42 @@ export const UnrecognizedRenderSnippetParams: LiquidCheckDefinition = {
       }
     }
 
+    function reportUnknownAliases(
```

Comment:
This is handled quite differently that Named Parameters so I split this into its own function